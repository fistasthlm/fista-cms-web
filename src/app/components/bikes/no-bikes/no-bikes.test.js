import React from 'react';
import { shallow } from 'enzyme';
import NoBikes from './no-bikes';

const setup = () => {
    return shallow(<NoBikes />);
};

describe('Render', () => {
    it('should render components', () => {
        const wrapper = setup();

        expect(wrapper).toMatchSnapshot();
    });
});
