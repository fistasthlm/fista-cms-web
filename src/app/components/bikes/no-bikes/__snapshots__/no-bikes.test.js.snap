// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render should render components 1`] = `
<div
  className="no-bikes"
>
  <h1>
    No bikes
  </h1>
  <span>
    add one for samlags  skull
  </span>
  <Link
    className="no-bikes__link"
    replace={false}
    to={
      Object {
        "pathname": "/add",
      }
    }
  >
    Add bieek
  </Link>
</div>
`;
